docs/source/markdown/podman-machine-inspect.1.md

```diff
@@ -32,6 +32,7 @@ Print results with a Go template.
 | .LastUp             | Time when machine was last booted                     |
 | .Name               | Name of the machine                                   |
 | .Resources ...      | Resources used by the machine                         |
+| .Rootful ...        | Machine rootful container execution                   |
```

Comment:
I find this hard to brain parse.  Can we elaborate a bit more what that means?

Reply:
This is my fault, but, could you remove the `...` please? `...` means "this is a format field with subfields that I really don't want to document". In this case `Rootful` is a simple bool.

Again, it's my fault for making this convention non-obvious and for not adding checks to the cross-ref script to whine about "hey don't use dot-dot-dot on this". I will fix that... sigh... some day. Thanks.

Reply:
haha @edsantiago i hear you on the "some day" !